torchdynamo/codegen.py

```diff
@@ -289,8 +328,36 @@ def make_call_generated_code(self, fn_name: str) -> List[Instruction]:
         self.extend_output(self.load_function_name(fn_name))
 
         graphargs = self.tx.output.graphargs
+
+        # to handle random calls
+        if len(self.tx.random_calls) > 0:
+            rand_fn_name = unique_id("__gen_rand_values")
+            rand_fn = torchdynamo.disable(_get_gen_rand_values_fn(self.tx.random_calls))
+            self.tx.output.install_global(rand_fn_name, rand_fn)
+            self.extend_output(self.load_function_name(rand_fn_name))
+            self.extend_output(
+                [
+                    create_instruction("CALL_FUNCTION", 0),
+                    self.create_store(self.tx.output.random_values_var),
```

Comment:
Currently it doesn't support graph break between multiple random calls, I'm thinking to store random values as global to support, which can be one of the follow-ups if it's necessary.